system/include/SDL/SDL_cpuinfo.h

```diff
@@ -105,21 +105,25 @@ extern DECLSPEC SDL_bool SDLCALL SDL_HasAltiVec(void);
  *  This function returns true if the CPU has MMX features.
  */
 extern DECLSPEC SDL_bool SDLCALL SDL_HasMMX(void);
+extern DECLSPEC SDL_bool SDLCALL SDL_HasMMXExt(void);
```

Comment:
What are these changes?

Reply:
These were previously declared in our JS library code but not in our header files.     I think perhaps in 1.2 they were added as `...Ext` and then in 1.3 (which is the weird version that doesn't seem to exist anywhere else but that we include the headers from) the `Ext` suffix was dropped.

I could instead delete from the JS implemenation?

Reply:
I see, sgtm to leave it as is.

Reply:
Actually the 1.2 headers on my system declare both:

```
/** This function returns true if the CPU has MMX features */                                                                      
extern DECLSPEC SDL_bool SDLCALL SDL_HasMMX(void);                               
                                                                                 
/** This function returns true if the CPU has MMX Ext. features */               
extern DECLSPEC SDL_bool SDLCALL SDL_HasMMXExt(void);    
```